runelite-client/src/test/java/net/runelite/client/util/ColorUtilTest.java

```diff
@@ -123,8 +126,66 @@ public void colorLerp()
 	public void colorToHexCode()
 	{
 		COLOR_HEXSTRING_MAP.forEach((color, hex) ->
-		{
-			assertEquals(hex, ColorUtil.colorToHexCode(color));
-		});
+			assertEquals(hex, ColorUtil.colorToHexCode(color)));
+	}
+
+	@Test
+	public void colorToAlphaHexCode()
+	{
+		COLOR_ALPHA_HEXSTRING_MAP.forEach((color, hex) ->
+			assertEquals(hex, ColorUtil.colorToAlphaHexCode(color)));
+	}
+
+	@Test
+	public void isHex()
+	{
+		COLOR_HEXSTRING_MAP.forEach((_color, hex) ->
```

Comment:
Iterate over `COLOR_HEXSTRING_MAP.values()` instead of key-value pairs if the key isn't used

Reply:
It would also be helpful to iterate over `COLOR_ALPHA_HEXSTRING_MAP` and test the expected result for those values, and vice-versa for the `isAlphaHex()` test following this.

Reply:
Done both suggestions; also added `INVALID_COLOR_HEXSTRING_LIST` constant with an assortment of strings that should fail both checks.